android/build.gradle

```diff
@@ -47,5 +47,5 @@ android {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    api 'io.sentry:sentry-android:6.25.1'
+    api 'io.sentry:sentry-android:6.26.0'
```

Comment:
Please cut a release that has this changes https://github.com/getsentry/sentry-java/pull/2857/files and upgrade the Android SDK

Reply:
Done in https://github.com/getsentry/sentry-react-native/pull/3170/commits/48d2ba958b466141b8b7cdfd9e872acac01d2b6f